docs/get-started/tooling/libraries/multicall.mdx

```diff
@@ -11,7 +11,7 @@ Multicall aggregates results from multiple contract constant function calls.
 This reduces the number of separate JSON RPC requests that need to be sent,
 which is especially useful if using remote nodes like Infura. This guarantees
 that all values returned are from the same block (like an atomic read) and
-returning the block number the values are from (giving them important context so
+returns the block number the values are from (giving them important context so
```

Comment:
```suggestion
return the block number the values are from (giving them important context so
```

Not sure this is necessary, as it changes the meaning. I believe the original sentence intends to list two conditions, i.e. that `all values returned` are both:
1. from the same block, and;
2. returning the block number the values are from. 

On that basis I think it would be best if we correct to `return` to make sure the verb conjugation matches the plural `values`.